pypesto/objective/objective.py

```diff
@@ -101,13 +101,18 @@ def __init__(self,
                  res=None, sres=None,
                  fun_accept_sensi_orders=False,
                  res_accept_sensi_orders=False,
+                 prior = None,
```

Comment:
That one looks not conformal with pep8
Run ./run_flake8.sh locally to get all formatting issues, fix them and repush ;)